modules/ROOT/pages/java.adoc

```diff
@@ -4,6 +4,7 @@
 :snippet: {examplesdir}/java/code-snippets/app/src/main/java/com/couchbase/code_snippets/MainActivity.java
```

Comment:
I would define this path without including `examplesdir`. Let Antora handle the location of the examples directory. See comments on `java.adoc` for more information.

```
:snippet: java/code-snippets/app/src/main/java/com/couchbase/code_snippets/MainActivity.java
```

Reply:
Ok. Didn't know that would work. Will do that from now and amend all the existing {examplesdir} as and when I find them. Like I say, just going along with what was there.